### DISCUSSION TITLE
The TheHive :Sorry, there are currently no analyzers for the selected observable type

### DISCUSSION THREAD

**Discussion by cqliukee** [Security-Onion-Solutions/securityonion#2509] [Unsupported Versions]
Observables  -->Add -->Type:hash.

select -->run Analyzers. The error information as following:

_Sorry, there are currently no analyzers for the selected observable type(s)_

**Comment by 0nnyx** [comment]
Go to the Cortex interface, Add a user & pass for the SecurityOnion organization with Roles: read, analyze, orgadmin.
Login back to Cortex with your created user, Click Organization tab at the top and you'll see Analyzers that you can configure & enable.

**Reply by cqliukee** [reply]
@0nnyx thanks very much. It can't work.
Full name: Lisa
Roles: read, analyze, orgadmin

![obervable](https://user-images.githubusercontent.com/48860749/103441926-7828bc00-4c8c-11eb-97bb-52bb969faf74.jpg)

**Reply by 0nnyx** [reply]
You're showing a new user created in Hive and not Cortex as I instructed...
Analyzers need to be enabled on Cortex using the new account you created then they are accessible in the Hive. So my only suggestion left is to read SecOnion & Cortex Docs.

**Reply by cqliukee** [reply]
@0nnyx Thanks very much.
I configured the analyzer in Cortex. E.g.:Virustotal, OTXQuery ,HybridAnalysis,Abuse_Finder.
![analyzer](https://user-images.githubusercontent.com/48860749/103450511-b0fb7c00-4cf2-11eb-9611-bdda56aaa29c.jpg)

I can find the following analyzer. 
Abuse_Finder_3_0
CERTatPassiveDNS_2_0 
![analyzer-1](https://user-images.githubusercontent.com/48860749/103450581-98d82c80-4cf3-11eb-84e3-0675b37fe4a7.jpg)

but can't find the Virustotal, OTXQuery ,HybridAnalysis analyers.

![no-analyzer](https://user-images.githubusercontent.com/48860749/103450586-a1306780-4cf3-11eb-84d8-2623f64398b0.jpg)

**Reply by weslambert** [reply]
Sounds like the sync b/w TheHive/Cortex is broken.  In TheHive, in the bottom-right corner of the screen -- does is the "brain"/Cortex icon have a green circle around it?
